### PR TITLE
feat: create Remember me feature

### LINKED ISSUE #6: chore: Add "Remember me" to the example
# Chore

## Describe the chore

- Add `Remember Me` functionality to the example: https://github.com/supabase/gotrue-js/blob/6d2ef91f738b8ee5339e4b3724f432acc1665ad2/example/src/App.js#L33

## Additional context

We should give a basic example of how to use enable/disable storing of sessions

### PR DESCRIPTION
**Pull request by niits**
## What kind of change does this PR introduce?

Add Remember me feature

I have read a few ways to implement the remember me function in several blog pages. Most of these blogs suggest saving username / email to localStorage. Others instruct to install with token but need backend installation so I chose the first option.

## What is the current behavior?

closes #6 

## What is the new behavior?

fdsaf

### CODE REVIEW

File: example/src/App.js [hunk 1]
```diff
@@ -6,8 +6,9 @@ const auth = new Client()
 
 function App() {
   let [session, setSession] = useState(null)
-  let [email, setEmail] = useState('')
+  let [email, setEmail] = useState(localStorage.getItem('email') ?? '')
   let [password, setPassword] = useState('')
+  let [remember_me, setRememberMe] = useState(false)
```
Comment (kiwicopple) [inline example/src/App.js:11]: One small request on style - can we please make this camel case? `rememberMe`

Comment (niits) [reply example/src/App.js:11]: ok

Comment (niits) [reply example/src/App.js]: @kiwicopple I have change this variable's name, Please check it out.

**Comment by kiwicopple** [comment]
Legend, thanks @niits 🚢

**Pull request merged**